.github/workflows/ubuntu.yml

```diff
@@ -105,11 +106,19 @@ jobs:
           git submodule update --init src/tests/resources/Antares_Simulator_Tests
 
     - name: Download XPRESS if needed
-      if: ${{ matrix.ortools-xpress == 'ON' }}
+      # if: ${{ matrix.ortools-xpress == 'ON' }}
       run: |
         git clone -b master https://${{ secrets.XPRS_USERNAME }}:${{ secrets.XPRS_TOKEN }}@github.com/rte-france/xpress-mp ${{ env.XPRESSDIR }}
         ls -la ${{ env.XPRESSDIR }}
 
+    - name: Configure compiler
+      run: |
+        if [ "${{ matrix.compiler }}" == "clang" ]; then
+          export CC=/usr/bin/clang
+          export CXX=/usr/bin/clang++
```

Comment:
Those env variables won't be available to the next steps.

See [here](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable).

Reply:
It only needs to be there for configuration, it's stored in the Cmake cache after that